templates/travel/elements/_typography.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2016 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Overide ampstart-base heading line-heights
+ */
+h1.line-height-2, h2.line-height-2, h3.line-height-2, h4.line-height-2, h5.line-height-2, h6.line-height-2,
+.h1.line-height-2, .h2.line-height-2, .h3.line-height-2, .h4.line-height-2, .h5.line-height-2, .h6.line-height-2 {
+  line-height: 1.4;
```

Comment:
this is bad, why only do it for line-height-2?, why not just set this as line-height-2 or , set this as the line height for all headings?

Reply:
I agree, I did it so that the basscss utility class would override the line-height property declared in `ampstart-base/elements/headings`

Reply:
if you want all your headings to have this line height, you can just override the H tags and classes.

Reply:
Will this work? https://github.com/ampproject/ampstart/pull/680/commits/d95840183c289e961dc495e92d6a5f537726c641

Reply:
why not just 
```css
h1, h2, h3, h4, h5, h6 {
  line-height: var(--line-height-2);
}
```

Reply:
same with the classes